clang/test/CodeGenCUDA/amdgpu-bf16.cu

```diff
@@ -111,19 +111,15 @@ __device__ __bf16 test_call( __bf16 in) {
 // CHECK-NEXT:    ret void
 //
 __device__ void test_vec_assign() {
-  typedef __attribute__((ext_vector_type(2))) __bf16 bf16_x2;
-  bf16_x2 vec2_a, vec2_b;
+  __bf16 [[clang::ext_vector_type(2)]] vec2_a, vec2_b;
```

Comment:
Would be nice to spell a pair of these the 'old' way as well (just as `__attribute__((ext_vector_type(2)))` to demonstrate that it works both ways).